cmd/postgres-operator/main.go

```diff
@@ -51,6 +52,11 @@ func initLogging() {
 }
 
 func main() {
+	// Set any supplied feature gates; panic on any unrecognized feature gate
```

Comment:
:memo: I put this first in `main` because misconfigurations will result in an abort, so I think it makes sense to fail quickly. That said, is there any reason this should go further down?